stestr/results.py

```diff
@@ -154,7 +154,7 @@ def _output_summary(self, run_id):
             values.append(('skips', skips, None))
         output.output_summary(
             not bool(failures), self._summary.testsRun, num_tests_run_delta,
-            time, time_delta, values)
+            time, time_delta, values, output=self.stream)
```

Comment:
I found that this is the missing part of output. The default of output is sys.stdout.